doc/minikube_setup.md

```diff
@@ -67,7 +67,7 @@ spec:
 ```
 
```

Comment:
This file has a broken link to tier2.md: https://github.com/pravega/pravega-operator/pull/512#diff-7a8fbea6e21112d3c6c13ab7cf4458dfbffcfd21a1bab49161cc18e30dc6817fR16

Reply:
fixed